src/NuGet.Clients/NuGet.SolutionRestoreManager/Resources.resx

```diff
@@ -124,8 +124,12 @@
     <value>Error occurred while restoring NuGet packages: {0}</value>
   </data>
   <data name="Error_PackageDownload_OnlyExactVersionsAreAllowed" xml:space="preserve">
-    <value>'{0}' is not an exact version like '[1.0.0]'. Only exact versions are allowed with PackageDownload.</value>
-    <comment>0 - the version string that's not exact</comment>
+    <value>Package '{1} {0}' does not have an exact version like '[1.0.0]'. Only exact versions are allowed with PackageDownload.</value>
```

Comment:
It'd be easier to read if this were 0, 1, in chronological order. Is there any reason for mapping the parameters as 1, 0?

Reply:
You are right. I have updated it now.